common/static/common/js/spec/components/paging_header_spec.js

```diff
@@ -0,0 +1,82 @@
+define([
+    'common/js/components/views/paging_header',
+    'common/js/components/collections/paging_collection'
+], function (PagingHeader, PagingCollection) {
+        'use strict';
+        describe('PagingHeader', function () {
+            var pagingHeader,
+                newCollection = function (size, perPage) {
+                    var pageSize = 5,
+                        results = _.map(_.range(size), function () { return {}; });
+                    var collection = new PagingCollection(
+                        {
+                            count: results.length,
+                            num_pages: results.length / pageSize,
+                            current_page: 1,
+                            start: 0,
+                            results: _.first(results, perPage)
+                        },
+                        {parse: true}
+                    );
+                    collection.start = 0;
+                    collection.totalCount = results.length;
+                    return collection;
+                };
+
+            it('can load templates', function () {
```

Comment:
What does ``it can load templates`` mean? That seems like an internal implementation detail. I'd probably remove this test since the next one also verifies that ``Currently viewing`` is rendered.